is_prime/script/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+version = "0.1.0"
+name = "fibonacci-script"
+edition = "2021"
+default-run = "fibonacci"
```

Comment:
Confusing..

Reply:
Yeah, see my answer below, the project generation harness is apparently bent on hard-coding Fibonacci everywhere.